README.md

```diff
@@ -127,40 +127,6 @@ make get-nodes-status
 watch -c -- make get-nodes-status
 ```
 
-### Image Region
-
-`update-operator.yaml` pulls operator images from the `us-west-2` region.
-To pull from a different region, replace the image endpoint to point at one of the following endpoints.
-This is not required, but pulling the image from the same region as your cluster will improve the download speed.
```

Comment:
Rather than deleting this entirely, I think it'd be better to leave in the regional repositories but change the way they're described here.  ECR Public isn't yet available through [PrivateLink](https://aws.amazon.com/privatelink/), so some people might still want to use the regional ECR repositories.